frontend-react/e2e/spec/all/public/developer-resources/api/documentation/index.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { DeveloperResourcesApiDocumentationPage } from "../../../../../../pages/public/developer-resources/api/documentation/index";
+import { test as baseTest, expect } from "../../../../../../test";
+
+export interface SecurityPageFixtures {
+    developerResourcesApiDocumentationPage: DeveloperResourcesApiDocumentationPage;
+}
+
+const test = baseTest.extend<SecurityPageFixtures>({
+    developerResourcesApiDocumentationPage: async (
+        {
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        },
+        use,
+    ) => {
+        const page = new DeveloperResourcesApiDocumentationPage({
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        });
+        await page.goto();
+        await use(page);
+    },
+});
+
+test.describe(
+    "Developer Resources / API / Documentation / Index page",
+    {
+        tag: "@smoke",
+    },
+    () => {
+        test("has correct title", async ({ developerResourcesApiDocumentationPage }) => {
+            await expect(developerResourcesApiDocumentationPage.page).toHaveTitle(
+                developerResourcesApiDocumentationPage.title,
+            );
+            await expect(developerResourcesApiDocumentationPage.heading).toBeVisible();
+        });
+
+        test("has side nav", async ({ developerResourcesApiDocumentationPage }) => {
+            await expect(
+                developerResourcesApiDocumentationPage.page.getByRole("navigation", { name: "side-navigation" }),
```

Comment:
Please check here for an example on how to use the new `.testSidenav()` method from `BasePage`: https://github.com/CDCgov/prime-reportstream/blob/2f0682e8e5a75ba4cfa7b59e0246560d54b989ad/frontend-react/e2e/spec/all/public/about/roadmap.spec.ts#L64

Reply:
This applies to `frontend-react/e2e/spec/all/public/developer-resources/api/documentation/responses-from-reportstream.spec.ts`, `frontend-react/e2e/spec/all/public/developer-resources/api/documentation/sample-payloads-and-output.spec.ts` and `frontend-react/e2e/spec/all/public/developer-resources/api/getting-started.spec.ts` as well.